app/javascript/react/components/molecules/Calendar/atoms/Month/Month.style.tsx

```diff
@@ -14,24 +14,25 @@ const Week = styled.div`
 const Month = styled.div`
   display: flex;
   flex-direction: column;
+  width: 100%;
 `;
 
 const MonthName = styled.span`
   font-size: 24px;
   font-weight: 600;
   color: ${gray400};
-  padding: 8px;
+  padding: 0.5rem;
   line-height: 120%;
   text-align: center;
   background-color: ${gray100};
   border-radius: 10px;
```

Comment:
We spoke about border radius in the Day and Month name component and were thinking it could stay. But seeing the smaller mobile month view I think it's easier to read when those two have rectangles - no border radius - I played with it in the browser. So I'd remove it for mobile in the end, as design proposes.

Reply:
Done